### DISCUSSION TITLE
How Do you make these mods

### DISCUSSION THREAD

**Discussion by willbillpill** [Woodi-dev/Among-Us-Sheriff-Mod#66] [Q&A]
I just wanted to know how to make mods like you and make my own mods if you can maybe you can tell me how to make my own mod

**Comment by KingsMMA** [comment]
Somebody has decompiled the code: https://github.com/Tobiti/Among-Us-Sheriff-Mod/tree/main/SheriffMod

You could kinda build of that if you knew how to compile it, but it'd be difficult.  Woodi doesn't seem to answer any questions like this or about compiling the source code

**Reply by Woodi-dev** [reply]
> Somebody has decompiled the code: https://github.com/Tobiti/Among-Us-Sheriff-Mod/tree/main/SheriffMod
> 
> You could kinda build of that if you knew how to compile it, but it'd be difficult. Woodi doesn't seem to answer any questions like this or about compiling the source code

It's a BepIneÉx Plugin. There are good guides about Harmony and BepInEx. Just follow their first instructions and you should know how to compile

**Reply by KingsMMA** [reply]
Hi,
I have had a look at BepInEx, and it does not work with il2cpp, which Among Us seems to use?

**Reply by Woodi-dev** [reply]
It does. It is called Bepinex bleeding edge...
[https://builds.bepis.io/projects/bepinex_be](https://builds.bepis.io/projects/bepinex_be)

**Reply by KingsMMA** [reply]
Thanks.
Is there a tutorial for bleeding edge?
https://bepinex.github.io/bepinex_docs/master/articles/dev_guide/plugin_tutorial/1_setup.html doesn't cover IL2CPP.

**Reply by willbillpill** [reply]
ok thanks

**Comment by Danial2k11134** [comment]
Do you have any step like download

**Comment by Danial2k11134** [comment]
Zzzzz

**Reply by Woodi-dev** [reply]
@Danial2k11134  check out Reactor Modding API if you wanna learn to mod
